bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html

```diff
@@ -30,8 +30,10 @@
 <nav id="health-subnav" class="health-subnav">
   <ul class="content">
     <li><a href="{{ url('mozorg.internet-health.privacy-security') }}">{{ _('Privacy &amp; Security') }}</a></li>
-    <li class="current"><a href="{{ url('mozorg.internet-health.digital-inclusion') }}">{{ _('Digital Inclusion') }}</a></li>
     <li><a href="{{ url('mozorg.internet-health.open-innovation') }}">{{ _('Open Innovation') }}</a></li>
+    <li><a href="{{ url('mozorg.internet-health.decentralization') }}">{{ _('Decentralization') }}</a></li>
```

Comment:
Good catch. Added the tag `health-decentralization-and-literacy` (just for the subnav on existing pages).